humio/resource_alert.go

```diff
@@ -87,6 +88,23 @@ func resourceAlert() *schema.Resource {
 				Optional: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+			"run_as_user_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"query_ownership_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				ValidateDiagFunc:	func(v interface{}, path cty.Path) diag.Diagnostics {
```

Comment:
A tab between `ValidateDiagFunc:` and `func`? Is this "right"?

Reply:
It is not! Good catch. Fixed.

Reply:
```suggestion
				ValidateDiagFunc: func(v interface{}, path cty.Path) diag.Diagnostics {
```